.github/workflows/platform.dependencies.yml

```diff
@@ -848,3 +856,73 @@ jobs:
           subscriptionId: '${{ secrets.ARM_SUBSCRIPTION_ID }}'
           managementGroupId: '${{ secrets.ARM_MGMTGROUP_ID }}'
           removeDeployment: '${{ env.removeDeployment }}'
+
```

Comment:
I'd expect some of the dependencies to leverage this kv as well in a next iteration. Should we move these 2 jobs (job_deploy_kv_platform and job_deploy_kv_tokens_secrets) right after the job_deploy_rg one?

Reply:
We can move them to the top, that's only visual though. My thinking would be to move this to the next iteration, as we need to think about building dependencies. The intent is to have this Key Vault as optional. So if something else from the dependency services 'depend' on it, then that's a change we need to think of. Thoughts?

Reply:
@eriqua I've moved the platform KV +  secrets job to the top right after RG.